ocaml/typing/parmatch.ml

```diff
@@ -1834,27 +1834,29 @@ let pressure_variants_in_computation_pattern tdefs patl =
 (* Utilities for diagnostics *)
 (*****************************)
 
+let is_guarded_rhs = function
```

Comment:
Can we change this to something like `is_fallthrough_possible` to avoid overloading "guarded".